runtime/near-wallet-contract/build.rs

```diff
@@ -41,23 +41,25 @@ fn build_contract(dir: &str, args: &[&str], output: &str) -> Result<()> {
 /// process, to avoid unexpected behaviors due to the workspace configurations.
 // TODO(eth-implicit) Change it to have a reproducible hash of the WASM file.
 // see https://github.com/near/nearcore/pull/10269#discussion_r1430139987.
```

Comment:
Can now remove this comment.

Reply:
Done in https://github.com/near/nearcore/pull/11011/commits/b279f67d496b449349e54042abfe13e9d383000f